clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/secured/FileJwtRetriever.java

```diff
@@ -23,19 +23,19 @@
 import java.nio.file.Path;
 
 /**
- * <code>FileTokenRetriever</code> is an {@link AccessTokenRetriever} that will load the contents,
+ * <code>FileJwtRetriever</code> is an {@link JwtRetriever} that will load the contents,
```

Comment:
```suggestion
 * <code>FileJwtRetriever</code> is a {@link JwtRetriever} that will load the contents,
```
(and are we missing something there? "contents of a file")

Reply:
Yep, not sure how long that's been missing a key part of the sentence 🤦‍♂️